process/css-styleguide/heading/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Heading</title>
+    <link href="./heading.css" rel="stylesheet" />
+    <link rel="stylesheet" href="../spacing.css" />
+  </head>
+  <body class="p-4">
+    <h1 class="typography title-xl">This is Title XL</h1>
+    <h2 class="typography title-l">This is Title L</h2>
+    <h3 class="typography title-m">This is Title M</h3>
+    <h4 class="typography title-s">This is Title S</h4>
+    <h5 class="typography title-xs">This is Title XS</h5>
+    <h6 class="typography mt-1 title-caps">This is Title CAPS</h6>
+    <h6 class="typography mt-1 title-caps-xs">This is Title CAPS XS</h6>
+
+    <br />
+    <br />
+    <br />
+
+    <div class="typography body-l">This is Body L - Default</div>
+    <div class="typography body-l text-bold">This is Body L - Bold</div>
+    <div class="typography body-l text-underline">This is Body L - Link</div>
+
+    <br />
+    <br />
+    <br />
+
+    <div class="typography mb-1 body-m">This is Body L - Default</div>
+    <div class="typography mb-1 body-m text-bold">This is Body L - Bold</div>
+    <div class="typography mb-1 body-m text-underline">
+      This is Body L - Link

```

Comment:
All the texts below are false:
<img width="265" alt="Capture d’écran 2022-07-13 à 10 18 21" src="https://user-images.githubusercontent.com/30866152/178685695-5ead966e-069a-430c-994d-f35ca6c52224.png">

They are specified as `body L` but they are not.